ecs-cli/modules/cli/compose/project/project_test.go

```diff
@@ -326,6 +329,23 @@ func setupTestProjectWithECSRegistryCreds(t *testing.T, ecsParamsFileName, credF
 	}
 }
 
+// This function mimics the implementation inside `project.NewProject()` for .env file lookup.
+// The following function is only used for unit tests.
+func getDefaultEnvironmentLookup() (*lookup.ComposableEnvLookup, error) {
```

Comment:
I might rename this to`mockGetDefaultEnvironment` or something to make it clear it's a testing utility function. 

Looking at what this replaces, I wonder if the reason `GetDefaultEnvironmentLookup()` existed in utils was mostly for testing.  In the past we've used monkey patching in tests (see [blahg](https://husobee.github.io/golang/testing/unit-test/2015/06/08/golang-unit-testing.html)) to mock out functions, which i guess this is doing though the actual lookup seems to be hidden.